pkg/operator/controller/canary/controller_test.go

```diff
@@ -145,3 +147,74 @@ func TestCycleServicePort(t *testing.T) {
 		}
 	}
 }
+
+func TestDeduplicateErrorStrings(t *testing.T) {
```

Comment:
*nit* I think we are trying to use `Test_[FunctionName]` in our unit tests.

Reply:
Is that a new initiative? From what I can tell we only do the `Test_[FunctionName]` syntax for tests of private functions, but I can change it if it's preferred for all new tests.

Reply:
Yea just checked - it was for all unit test functions, regardless of the scope of the function.